migrations/kdvh/dump/dump.go

```diff
@@ -19,8 +20,14 @@ import (
 // List of columns that we do not need to select when extracting the element codes from a KDVH table
 var INVALID_COLUMNS = []string{"dato", "stnr", "typeid", "season", "xxx"}
 
-func DumpTable(table *db.Table, pool *pgxpool.Pool, config *DumpConfig) {
-	if err := os.MkdirAll(filepath.Join(config.BaseDir, table.Path), os.ModePerm); err != nil {
+func DumpTable(table *db.Table, pool *pgxpool.Pool, config *Config) {
+	fmt.Printf("Dumping %s...\n", table.TableName)
+	defer func() {
+		fmt.Println(strings.Repeat("- ", 50))
+		log.SetOutput(os.Stdout)
+	}()
```

Comment:
What is going on here?

Reply:
This is just to get pretty output in the terminal, but now that I think about I probably don't need the `log.SetOutput` since that's anyway changed at the start of this function

Reply:
Yeah, `log.SetOutput` was the part I didn't understand